internal/service/project/resource_project_test.go

```diff
@@ -895,71 +884,6 @@ func createDataFederationLimit(limitName string) admin.DataFederationLimit {
 	}
 }
 
-type MockProjectService struct {
-	mock.Mock
-	MockResponses []ProjectResponse
```

Comment:
I believe the ProjectResponse struct could be removed right? Each test case can define the specific SDK model it will return.

Reply:
Done, defining one struct for each specific SDK model adds a bit more of code, but makes it clearer.